javaparser-core-testing/src/test/java/com/github/javaparser/printer/lexicalpreservation/TransformationsTest.java

```diff
@@ -36,7 +36,7 @@ public void unchangedComplexFile() throws IOException {
     @Test
     public void example1() throws IOException {
         considerExample("Example1_original");
-        cu.getClassByName("A").get().getFieldByName("a").get().setModifiers(EnumSet.of(Modifier.STATIC));
+        cu.getClassByName("A").get().getFieldByName("a").get().addModifier(STATIC);
```

Comment:
add seems semantically different from set but I will assume that in this case this does not matter